### PR TITLE
Fix rolling redirection

### LINKED ISSUE #116: Fix rolling node regex

### PR DESCRIPTION
**Pull request by louptheron**
Fix https://github.com/octo-technology/tezos-link/issues/116

### CODE REVIEW

File: data/proxy/conf/prod.toml [hunk 1]
```diff
@@ -29,4 +29,5 @@ routineDelaySeconds = 60
 whitelistedRolling = [
   "(.*?)/head/(.*?)",
   "(.*?)/head",
+  "(.*?)/injection/operation",
```
Comment (aboulay) [inline data/proxy/conf/prod.toml:32]: Why ?

File: internal/proxy/domain/model/node_type.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package model
+
+// NodeType represents the int
+type NodeType int
+
+// ArchiveNode and RollingNode node types
+const (
+    ArchiveNode NodeType = iota + 1
```
Comment (aboulay) [inline internal/proxy/domain/model/node_type.go:8]: The syntax is a bit strange... Is it normal ?

File: data/proxy/conf/prod.toml (hunk 1 above)
Comment (louptheron) [reply data/proxy/conf/prod.toml:32]: To allow users to inject transactions

File: internal/proxy/domain/model/node_type.go (hunk 2 above)
Comment (louptheron) [reply internal/proxy/domain/model/node_type.go:8]: Yes, it's Go's syntax for enum

Comment (aboulay) [reply internal/proxy/domain/model/node_type.go:8]: Ok ugly but good enough. I approve the PR

**Review by aboulay — APPROVED** [review]
Thanks for your work :)

**Pull request merged**